scripts/levitate-show-affected-plugins.js

```diff
@@ -13,8 +13,12 @@ const { execSync } = require('child_process');
  * @returns {string} - The extracted package name, or an empty string if no match is found.
  */
 function getPackage(location) {
-  const match = location.match(/\/(@[^@]+)@/);
-  return match ? match[1] : '';
+  const match = location.match(/\/(grafana-[^/]+)\/dist/);
```

Comment:
are you sure we want to tie this to packages called "grafana-?"

Reply:
This seems to be the pattern that the `location` is following right?
https://github.com/grafana/grafana/pull/91164#issuecomment-2258788368
![location-print](https://github.com/user-attachments/assets/d4ae1ada-482b-4333-a075-b22eab00d927)


The [script](https://github.com/grafana/grafana/blob/main/scripts/check-breaking-changes.sh#L4) is iterating over the directory names that also follow this pattern after we decompress them:
https://github.com/grafana/grafana/tree/main/packages

Or is there something that I am missing?